doc/releases/release-notes-4.1.rst

```diff
@@ -101,6 +101,7 @@ Boards & SoC Support
 
   * All HWMv1 board name aliases which were added as deprecated in v3.7 are now removed
     (:github:`82247`).
+  * Enabled USB on NXP ``frdm_mcxc444``
```

Comment:
Please remove this change as the whole release notes for 4.1 is under discussion about needs to be included.

Reply:
A rebase is required on this PR. So might as well remove it.